epitator/geoname_classifier.py

```diff
@@ -12,46 +12,48 @@
 {'C': 1.0,
  'class_weight': None,
  'classes_': array([False,  True], dtype=bool),
- 'coef_': array([[ 0.30886707,  0.02116973,  0.00560847,  0.27765891,  1.94456142,
-         0.52508845,  0.18140931,  0.        , -0.95166464,  0.01895766,
-        -0.01038021, -0.7187989 , -0.24003574,  2.73676538,  0.        ,
-        -0.12539797,  0.        ,  0.        ,  0.        ,  0.        ,
+ 'coef_': array([[ 0.3213819 ,  0.01823242,  0.02336938,  0.29849362,  2.47665562,
+         0.47210803,  0.16109926,  0.        , -0.97918471,  0.00476336,
+        -0.01442839, -0.60016267, -0.23542543,  1.89213763,  0.        ,
+        -1.68017007,  0.        ,  0.        ,  0.        ,  0.        ,
          0.        ]]),
```

Comment:
How did we determine these new values?

Reply:
The training script at https://github.com/ecohealthalliance/geoname-annotator-training

Reply:
Do we know what effect this had on annotator performance, or are we not doing cross-validation on the GeoName annotator?

Reply:
There's no cross validation, but the geoname annotator is scored against the training data. There appears to be a drop in accuracy. I think it's worth taking since we'll be on stronger footing moving forward with SpaCy 2. Here's the current and previous stats:

```
current:
tps: 1223
fns: 451
fps: 487
ignored: 247

previous:
tps: 1272
fns: 402
fps: 425
ignored: 246
```